src/main/resources/openshift/builder/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jboss-eap-7/eap70-openshift:1.4
+FROM jboss-eap-7/eap70-openshift:latest
```

Comment:
Ah, I see. I didn't realize we do image naming and versioning little bit differently :) In this case version is (sort of) part of image name, so I guess it will always have EAP 7.0.x.

So in this case it is OK to depend on latest. Sorry, I didn't realize that.